.wp-env.json

```diff
@@ -3,6 +3,7 @@
 	"plugins": [
 		"https://downloads.wordpress.org/plugin/woocommerce.latest-stable.zip",
 		"https://github.com/WP-API/Basic-Auth/archive/master.zip",
+		"https://github.com/nielslange/woo-test-helper/archive/refs/heads/trunk.zip",
```

Comment:
Should we move this zip to our repository?
Like under woocommerce-gutenberg-products-block/tests/utils/?